_includes/manuals/3.12/4.3.3_smartproxy_bmc.md

```diff
@@ -8,8 +8,9 @@ Activate the BMC management module within the Smart Proxy instance.  This allows
 
 Available providers are:
 
-* `freeipmi` - for IPMI control using the freeipmi implementation
-* `ipmitool` - using the ipmitool implementation
+* `freeipmi` - for IPMI control using the freeipmi implementation. Requires [freeipmi](https://www.gnu.org/software/freeipmi) and [rubyipmi](https://rubygems.org/gems/rubyipmi) Ruby gem.
+* `ipmitool` - using the ipmitool implementation. Requires [ipmitool](https://github.com/ipmitool/ipmitool) and [rubyipmi](https://rubygems.org/gems/rubyipmi) Ruby gem.
```

Comment:
The installer should install these if you pass `--foreman-proxy-bmc-default-provider`:
https://github.com/theforeman/puppet-foreman_proxy/blob/b73685cebc4efad1c23844843c0f98e1e6454138/manifests/install.pp#L18-L20

Also, the gem is pulled in by the foreman-proxy RPM package:
https://github.com/theforeman/foreman-packaging/blob/0dc7a7e095c9d9893236417c0a64fafa6ee05c13/packages/foreman/foreman-proxy/foreman-proxy.spec#L46

With Debian it's in `Recommends`, but that's also true for `ruby-redfish-client`.
https://github.com/theforeman/foreman-packaging/blob/cdd1c1f7e9af04ca10aa645c52e1ef45cf7bca5f/debian/bookworm/foreman-proxy/control#L16

Reply:
Done